spec/lib/index.spec.js

```diff
@@ -0,0 +1,219 @@
+"use strict";
+
+describe("schema", () => {
+    var mock, path, promiseMock, SchemaValidator, schemaValidatorInstance;
+
+    beforeEach(() => {
+        var fs, tv4;
+
+        fs = require("fs");
+        mock = require("mock-require");
+        path = require("path");
+        spyOn(path, "resolve").and.callThrough();
+        promiseMock = require("../mock/promise-mock")();
```

Comment:
I am actually unsure. I was following OpenToken's example. Removing it though. ac79550